ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activityTextAnswer.component.ts

```diff
@@ -77,9 +78,18 @@ export class ActivityTextAnswer implements OnChanges, OnInit {
     @Output() valueChanged: EventEmitter<string | null> = new EventEmitter();
     public inputSubject = new Subject<string>();
     public filteredSuggestions$: Observable<TextSuggestion[]>;
+    public textInputModelChanged$ = new Subject<string>();
 
     public ngOnInit(): void {
         this.initFilteredSuggestions();
+
+        this.textInputModelChanged$.pipe(
+            debounceTime(300),
+            distinctUntilChanged()
+            )
+            .subscribe((value: string) => {
```

Comment:
Might have to handle unsubscribe

Reply:
fixed at 9be26f0d422939a3407d3a6e149be665068dd7cc